tests/unit/Makefile.am

```diff
@@ -128,6 +129,11 @@ test_message_SOURCES         = test_platform.cpp test_message.cpp
 test_ncp_buffer_LDADD        = $(COMMON_LDADD)
 test_ncp_buffer_SOURCES      = test_platform.cpp test_ncp_buffer.cpp
 
+test_settings_LDADD          = $(COMMON_LDADD)  \
+                               $(top_builddir)/examples/platforms/posix/libopenthread-posix.a \
+                               $(NULL)
+test_settings_SOURCES        = test_settings.cpp
```

Comment:
But till my understanding, it is almost the same as what posix does except that using in-memory space but not flash?